### PR TITLE
Hot-fix: handle error instead of throwing it

### PR DESCRIPTION
**Pull request by zo-el**
- Updated the holo-nixpkgs 
- handle errors in the list or transactions.

### CODE REVIEW

File: src/data-interfaces/HoloFuelDnaInterface.js [hunk 1]
```diff
@@ -249,7 +248,7 @@ const HoloFuelDnaInterface = {
       const params = since ? { since } : {}
 
       const { transactions } = await createZomeCall('transactions/list_transactions')(params)
-      const nonActionableTransactions = transactions.map(presentTransaction)
+      const nonActionableTransactions = transactions.map(presentTransaction).filter(tx => !(tx instanceof Error))
```
Comment (robbiecarlton) [inline src/data-interfaces/HoloFuelDnaInterface.js]: think you want to filter the list before mapping `presentTransaction` over it. `presentTransaction(anError is not defined)`

Comment (robbiecarlton) [inline src/data-interfaces/HoloFuelDnaInterface.js:251]: think you want to filter the list before mapping `presentTransaction` over it. `presentTransaction(anError)` is not defined

**Review by robbiecarlton — CHANGES_REQUESTED** [review]

File: src/data-interfaces/HoloFuelDnaInterface.js (hunk 1 above)
Comment (robbiecarlton) [reply src/data-interfaces/HoloFuelDnaInterface.js:251]: obv, same feedback everywhere you're using this pattern

Comment (zo-el) [reply src/data-interfaces/HoloFuelDnaInterface.js:251]: @robbiecarlton, I dont think I understand. 
According to me the map build the array with the errors and then we filter it.

Comment (robbiecarlton) [reply src/data-interfaces/HoloFuelDnaInterface.js:251]: My mistake. I misread it, you're right.

**Review by robbiecarlton — APPROVED** [review]

**Comment by robbiecarlton** [comment]
@zo-el LGTM

**Review by JettTech — APPROVED** [review]

**Pull request merged**